content/blog/2023-08-22-keda-cncf-graduation.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "KEDA is gradutating to CNCF Graduted project 🎉"
+date = 2023-08-22
+author = "KEDA Maintainers"
+aliases = [
+"/blog/keda-cncf-incubation"
+]
++++
+
+In 2019, KEDA embarked on a mission to make application autoscaling on Kubernetes dead-simple. Our aim was to make sure that every Kubernetes platform can use it to scale applications without having to worry about the underlying autoscaling infrastructure.
+
+As part of that mission, we wanted to build a vendor-neutral project that is open to everyone and nicely integrates with other tools. Because of that, the KEDA maintainers decided that the Cloud Native Computing Foundation (CNCF) was a natural fit and got accepted as a sandbox project in 2020.
+
+KEDA started evolving and became a more mature autoscaler which came with more scalers out-of-the-box and introduced new concepts such as scaled jobs, trigger authentication, operational metrics, and more. The community started growing as well, the project governance matured and KEDA became a CNCF Incubation project in 2021.
+
+In the past year and a half, KEDA started focussing on making it easier to use for larger enterprises and use KEDA at-scale serving thousands of scaled objects with low performance impact.
+
+It has been great to see how KEDA has been adopted and is being used by more than 45+ listed end-users:
+
+![architecture diagram](/img/blog/keda-cncf-graduation/keda-graduation-end-users.gif)
+
+Today is another important milestone in KEDA's journey - **We are excited to announce that the [CNCF has accepted to move KEDA to a CNCF Graduated project](https://www.cncf.io/announcements/2023/08/22/cloud-native-computing-foundation-announces-graduation-of-kubernetes-autoscaler-keda/) next to projects such as Kubernetes, Prometheus, Istio and others! 🎉.**
```

Comment:
It might be better to shift the ordering here:
```suggestion
Today is another important milestone in KEDA's journey - **We are excited to announce that the [CNCF has accepted to move KEDA to a CNCF Graduated project](https://www.cncf.io/announcements/2023/08/22/cloud-native-computing-foundation-announces-graduation-of-kubernetes-autoscaler-keda/) next to projects such as Kubernetes, Prometheus, Istio and others! 🎉.**

It has been great to see how KEDA has been adopted and is being used by more than 45+ listed end-users:

![architecture diagram](/img/blog/keda-cncf-graduation/keda-graduation-end-users.gif)
```